### PR TITLE
[13.0][MIG] configuration_helper

### PR DESCRIPTION
**Pull request by phuctranfxvn**

### CODE REVIEW

File: configuration_helper/i18n/configuration_helper.pot [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Translation of Odoo Server.
```
Comment (i-vyshnevska) [inline configuration_helper/i18n/configuration_helper.pot:1]: you can remove this file

**Review by i-vyshnevska — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

File: configuration_helper/i18n/configuration_helper.pot (hunk 1 above)
Comment (simahawk) [reply configuration_helper/i18n/configuration_helper.pot:1]: pot file commits can be trashed indeed

**Comment by grindtildeath** [comment]
Can we maybe have a test module for that? Did anyone else test it?

I tried to test it but it seems that fields are not created properly in res.config.settings :confused:

**Pull request merged**